common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/workbench/PartHelper.xtend

```diff
@@ -1,9 +1,17 @@
 package org.testeditor.dsl.common.ui.workbench
 
+import org.eclipse.core.resources.IFile
+import org.eclipse.ui.IEditorPart
 import org.eclipse.ui.PlatformUI
+import org.eclipse.ui.part.FileEditorInput
 
 class PartHelper {
 
+	def IEditorPart findEditor(IFile input) {
```

Comment:
Would be nicer to use `Optional` here to point out that the result might be `null`.